ckan/cli/generate.py

```diff
@@ -77,3 +80,44 @@ def extension(output_dir):
 
     cookiecutter(template_loc, no_input=True, extra_context=context,
                  output_dir=output_dir)
+
+
+@generate.command(name=u'config', short_help=u'Create config from template.')
+@click.option(u'-o', u'--output-dir', help=u'Location to put the generated '
+                                           u'template.')
+def make_config(output_dir):
+    if output_dir is None:
+        print('\nERROR: Try again with \'-o path/to/ckan.ini\'')
+        sys.exit(1)
+
+    name = 'ckan.ini'
+
+    if name not in output_dir:
+        print('\nERROR: Config file must be named "ckan.ini"')
+        sys.exit(1)
+
+    # Output to current directory if no path is specified
+    if output_dir == name:
+        output_dir = os.getcwd() + '/' + output_dir
+
+    cur_loc = os.path.dirname(os.path.abspath(__file__))
+    os.chdir(cur_loc)
+    os.chdir('../config')
+    template_loc = os.getcwd() + '/deployment.ini_tmpl'
```

Comment:
This can be accessed more concisely with 
```python
os.path.join(cur_loc, '..', 'config', 'deployment.ini_tmpl')
```

Reply:
That's much better. I always forget about `path.join`...